docs/production-deployment/cloud/api-keys.mdx

```diff
@@ -196,6 +196,298 @@ tcld apikey disable --id <api-key-id>
+
+#### Overview
+
+Temporal Cloud API Keys facilitate identity-level authentication, for Users and Service Accounts. Each key is unique to a single identity and ties directly to the identity's Role-Based Access Control (RBAC) permissions, ensuring secure and appropriate access.
+
+The process is as follows:
+
+API Key (authentication) → User or Service Account (identity) → RBAC (authorization)
+
+These API Keys are used to authenticate Temporal Cloud operator tools, including Temporal Cloud CLI (tcld) and the Temporal Cloud Operations API (Cloud Ops API), and Client Connection tools, including the Temporal CLI and SDKs.
+
+
+#### API Key Summary
+
+Temporal Cloud API Keys functionality is designed for 3 types of use:
+
+1. User created API Keys - if enabled, users can create, delete, and update their own API Keys through the Cloud UI or tcld.
+2. Service Account created API Keys - Global Administrators can create, delete, and update API Keys for Service Accounts. 
+3. API Key governance - Temporal Cloud provides affordances for Global Administrators to govern and control API Key use within their accounts. Global Administrators can enable/disable API Keys for their accounts, and disable/delete API Keys for any User and/or Service Account.
+
+By default, API Keys must be enabled by a Global Administrator before use.
+
+#### Get started
+
+Prerequisites
+
+- Access to the Temporal Cloud UI or Temporal Cloud CLI (tcld) - to create an API Key
+- Enable access to API Keys for your Account
+- Create a new Namespace that allows API Key connectivity.
+
+#### Temporal CLI
+
+To use your API Key with the Temporal CLI, you can pass in the API Key to each command using the `--api-key` flag or an environmental variable (best practice).
+
+In addition to passing in the API Key, the following client options are required:
+
+- `--address`: Provide the Namespace's gRPC endpoint located in the Namespace UI in the gRPC endpoint box.
+  - For API Key connectivity, the endpoint format is `<region>.<cloud provider>.api.temporal.io:7233`
+  - Can be set using an environmental variable
+- `--namespace`: Provide the namespace.accountId that appears at the top of the Namespace page in the UI.
+  - Format is `<namespace.accountId>`
+  - Can be set using an environmental variable
+- `--grpc-meta "temporal-namespace="`: Provide the namespace.accountId again as grpc-metadata.
+- `--tls` for a secure connection with the appropriate options
+  - Potentially, with `--tls-disable-host-verification` to disable host verification
+  - Can be set using an environmental variable
+
+Here's an example of connecting to Temporal Cloud from the CLI using an environmental variable for the API Key:
+
+```bash
+export TEMPORAL_API_KEY=<key secret>
+temporal workflow list --address <endpoint> \
+--namespace <namespace.accountid> \
+--tls --tls-disable-host-verification \
+--grpc-meta "temporal-namespace=<namespace.accountid>"
+```
+
+#### SDK
```

Comment:
i'll defer to the SDK team, but it seems fine at first glance.

Reply:
These look good to me except the Java one is missing the more convenient `addApiKey` API that was added in https://github.com/temporalio/sdk-java/pull/2052. The samples should show that as well as the more manual method like the other languages do.

Reply:
`addApiKey ` is not released yet FWIW